packages/run-protocol/src/reserve/assetReserve.js

```diff
@@ -95,6 +96,36 @@ const start = async (zcf, privateArgs) => {
   /** @type {ZCFMint} */
   const runMint = await zcf.registerFeeMint('RUN', feeMintAccess);
 
+  const { brand: runBrand } = await E(runMint).getIssuerRecord();
+  const { publication: metricsPublication, subscription: metricsSubscription } =
+    makeSubscriptionKit();
+
+  // shortfall in Vaults due to liquidations less than debt. This value can be
+  // reduced by various actions which burn IST.
+  let vaultShortfall = AmountMath.makeEmpty(runBrand);
+
+  const updateMetrics = () => {
+    const metrics = harden({
+      allocations: getAllocations(),
+      shortfall: vaultShortfall,
+    });
+    metricsPublication.updateState(metrics);
+  };
+  updateMetrics();
+
+  const addLiquidationShortfall = shortfall => {
```

Comment:
for parity with `reduce…`
```suggestion
  const increaseLiquidationShortfall = shortfall => {
```

Reply:
done